database/migrations/2017_06_05_053734_create_category_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCategoryTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('categories', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('nama', 55);
+            $table->string('deskripsi', 255);
```

Comment:
mohon tambahkan method `nullable()` pada field ini karena deskripsi kategori biasanya boleh kosong. Kemudian untuk penamaan field usahakan in english ya, hal tersebut juga berlaku untuk penamaan variable, method, class, dll. cc @devinlukianto @kharisisriyant